app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java

```diff
@@ -42,12 +46,14 @@
      *
      * @param itemView Nonnull inflated view.
      */
-    public FaqViewHolder(@NonNull final View itemView, SparseBooleanArray sba) {
+    public FaqViewHolder(@NonNull final View itemView, SparseBooleanArray sba, Context mContext) {
         super(itemView);
         ButterKnife.bind(this, itemView);
         this.sba = sba;
         sba.put(getAdapterPosition(), false);
-        itemView.setOnClickListener(v -> showAnswer());
+        setCopyListener(questionTV,mContext);
+        setCopyListener(answerTV,mContext);
```

Comment:
no need to pass activity context all the way here. We can do `itemView.getContext()`

Reply:
I think it's hard to long press questionTv avoiding answerTv (and vice versa). Better put single copyListener and copy both question and answer  - maybe something like
```
[Q] What is the question?
[A] Anyway, this is answer.
```

Reply:
Ok, Will copy both question and answer together.